specification.md

```diff
@@ -4768,6 +4810,25 @@ States that are marked as `usedForCompensation` can define [error handling](#Wor
 (errors not explicitly handled),
 workflow execution should be stopped, which is the same behavior as when not using compensation as well. 
 
+### Workflow Versioning
+
+In any application, regardless of size or type, one thing is for sure: changes happen.
+Versioning your workflow definitions is an important task to consider. Versions indicate 
+changes or updates of your workflow definitions to the associated execution runtimes. 
+
+There are two places in the [workflow definition](#Workflow-Definition-Structure) where versioning can be applied:
+
+1. Top level workflow definition `version` property.
+2. Actions [subflowRef](#SubFlowRef-Definition) `version` property.
+
+The Serverless Workflow specification does not mandate a specific versioning strategy
+for the top level and actions subflowRef definitions `version` properties. It does not mandate the use 
+of a versioning strategy at all. We do recommend however that you do use a versioning strategy 
+for your workflow definitions especially in production environments. 
+
+To enhance portability when using versioning of your workflow and sub-workflow definitions,
```

Comment:
As far as I can tell, so far the spec is avoiding any assumptions on the versioning scheme.   How does the users choice of versioning scheme affect portability if runtimes are supposed to be agnostic to the versioning scheme?

Reply:
i'm not really sure i get the question.

Reply:
I'm trying to understand how the users choice of versioning scheme's affects portability, I was hoping you could enlighten me on this.

Reply:
@tsurdilo Not sure if you missed this question.

Reply:
The idea in the text is that if you use a standard such as SemVer rather than some custom versioning scheme, there is a higher probability chance it will be supported by more runtimes